src/test/java/core/basesyntax/dao/WarehouseDaoWriterTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class WarehouseDaoWriterTest {
+    public static final String PATH_TEST_FILE = "src/test/resources/output.csv";
+    private static final WarehouseDaoWriter warehouseDaoWriter = new WarehouseDaoWriter();
+
+    @Test
+    void writeData_normalBehavior_ok() {
+        Map<String, Integer> fruitMap = new HashMap<>();
+        fruitMap.put("banana", 20);
+        fruitMap.put("apple", 42);
+        warehouseDaoWriter.writeData(PATH_TEST_FILE, fruitMap);
+        String actualReport = "";
+        try {
+            actualReport = String.join(
+                    System.lineSeparator(), Files.readAllLines(Path.of(PATH_TEST_FILE)));
+        } catch (IOException e) {
+            throw new RuntimeException("File not found", e);
+        }
```

Comment:
Try/catch structure is rarely required in tests. You may just put throws to method header and if it occurs you will see that exception in test report